doc/nrf/libraries/security/trusted_storage.rst

```diff
@@ -84,7 +84,11 @@ Before using the trusted storage library with its default settings and options,
 * Zephyr's settings subsystem has to be enabled for use by setting the Kconfig option :kconfig:option:`CONFIG_SETTINGS`.
 
   * The settings subsystem uses the :ref:`zephyr:nvs_api` file system by default.
```

Comment:
Actually there is no default. It's the configuration of the application that will determine that. And from now on, it will default to ZMS if ZMS is enabled, and then to NVS if NVS is enabled (and ZMS is not).